src/prefect/runner/storage.py

```diff
@@ -564,6 +569,140 @@ def __eq__(self, __value) -> bool:
         return False
 
 
+class ModuleFunction:
```

Comment:
I think we'd be better off adding support for module path entrypoints to deployments. We'd be able to remove the need to copy the flow to a temporary directory. I can spend some time today spiking the feasibility of module path entrypoints.

Reply:
I like this